Package.resolved

```diff
@@ -1,14 +1,5 @@
 {
   "pins" : [
-    {
-      "identity" : "accessibilitysnapshot",
-      "kind" : "remoteSourceControl",
-      "location" : "https://github.com/EmergeTools/AccessibilitySnapshot.git",
-      "state" : {
-        "revision" : "54046d8fa44b9fa9f0e2229526f6ed89cb5e0ec2",
-        "version" : "1.0.2"
-      }
-    },
```

Comment:
Is this intentional?

Reply:
xcode did this automatically when updating `snapshotpreviews` to `0.10.22`

Reply:
ok, I removed this and it is still working, so I think it's ok to remove 😄